R/transformCounts.R

```diff
@@ -337,7 +337,7 @@ setMethod("transformFeatures", signature = c(x = "SummarizedExperiment"),
              pseudocount = 0,
              ...){
 
-        .Deprecated("transformCounts")
+        .Deprecated("transformAssay")
```

Comment:
Make sure that the `transformCounts` is still available if users call that but it should throw the informative warning. You can implement transformCounts as a wrapper that directly calls transformAssay after throwing the warning. I think there are some other examples of such function deprecation.

Make sure that this generates an informative error message, suggesting to change the function name in user workflows. You can check the other deprecated functions for examples on what kind of warnings they throw.